### DISCUSSION TITLE
Is that trivy having Capability to scan and sandbox container images ?

### DISCUSSION THREAD

**Discussion by mightysai1997** [aquasecurity/trivy#6131] [Q&A]
### Question

Is that trivy having Capability to scan and sandbox container images upon download from 3rd parties to analyses prior to developing apps on it.  
Malicious Behavior i.e. malware, crypto miners and outbound C2

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

**Comment by knqyf263** [comment]
No, Trivy doesn't have it.
https://github.com/aquasecurity/trivy/blob/main/docs/community/principles.md

**Comment by itaysk** [comment]
FYI, while not in Trivy Open Source, we do have it as a commercial offering from Aqua https://www.aquasec.com/products/container-analysis/
